### PR TITLE
Design update v3

### PR DESCRIPTION
**Pull request by dashouse**
WIP for updating Design System to new styles in GOV.UK Frontend v3

### CODE REVIEW

**Comment by hannalaakso** [comment]
Looks good @dashouse 👍 Noticed that the tab headings when they're "current" but not focused (because they're open on page load) look maybe a bit odd as they're black and without the underline 
Screen Shot 2019-06-20 at 13 07 08 
Also, the mobile menu button has poor contrast when it's focused.

**Comment by dashouse** [comment]
That's the case for tabs component too so I think it's ok. 

For non-text contrast purposes it can't have an underline because it needs to be different from the unselected tab because the border doesn't meet contrast.

Good call on mobile styles, I'm not actually sure I looked at the mobile nav at all yet

File: src/javascripts/components/mobile-navigation.js [hunk 1]
```diff
@@ -45,12 +45,15 @@ MobileNav.prototype.bindUIEvents = function () {
...
         $nextSubNav.classList.remove(subNavActiveClass)
         $toggler.classList.remove(subNavTogglerActiveClass)
+        $toggler.parentNode.classList.remove("app-mobile-nav__subnav-toggler-parent--active")
 
         $nextSubNav.setAttribute('aria-hidden', 'true')
         $toggler.setAttribute('aria-expanded', 'false')
       } else {
         $nextSubNav.classList.add(subNavActiveClass)
         $toggler.classList.add(subNavTogglerActiveClass)
+        console.log($toggler.parentNode)
```
Comment (aliuk2012) [inline src/javascripts/components/mobile-navigation.js]: Need to remove this

File: src/stylesheets/components/_site-search.scss [hunk 2]
```diff
@@ -78,7 +78,7 @@ $icon-size: 40px;
     width: 100%;
     margin-left: 0;
     outline: 3px solid $govuk-focus-colour;
-    outline-offset: 0;
+    outline-offset: -2px;
```
Comment (36degrees) [inline src/stylesheets/components/_site-search.scss]: @dashouse do you think we should make these changes as part of the focus state work? Wonder if it'd make sense to look at it separately alongside #937?

Comment (dashouse) [reply src/stylesheets/components/_site-search.scss]: Just the line?

**Comment by 36degrees** [comment]
Closing this now as I believe the changes have been made in:

- Update navigation focus styles #977 
- Update the header #978 
- Update examples to use new design and focus states #983 
- Design tweaks #989

**Pull request closed without merging**